resources/views/store/manage_vouchers.blade.php

```diff
@@ -270,6 +271,21 @@ function () {
                     collectedOn.datepicker({dateFormat: 'dd-mm-yyyy'}).val();
                 }
                 collectedOn.valueAsDate = new Date();
+
+                $("#collected-on").change(function() {
+                  var chosenDate = $(this).val();
+                  var chosenDateJS = new Date(chosenDate);
+                  var sixWeeks = new Date();
+                  sixWeeks.setDate(sixWeeks.getDate() + 42);
+                  if (chosenDateJS > sixWeeks) {
+                    $('#dateError').text('Please choose a date within six weeks of today');
+                    $('#dateError').css('font-size', '12px');
+                    $('#dateError').css('color', 'red');
+                    $('#dateError').show();
+                  } else {
+                    $('#dateError').hide();
```

Comment:
`display:none` is different to `display:hidden`; does `hide()` actually do `display:none`?